src/main/kotlin/com/github/etkachev/nxwebstorm/utils/GenerateTerminalCommand.kt

```diff
@@ -20,7 +20,8 @@ fun getSchematicCommandFromValues(
     }
     finalText
   }
-  val prefix = if (type == "workspace-schematic") "nx workspace-schematic $id" else "nx generate $type:$id"
+  val nx = "node node_modules/@nrwl/cli/bin/nx.js";
```

Comment:
kotlin linter says no semi colons..... yea it took a bit to get used to for me as well 😆 

```suggestion
  val nx = "node node_modules/@nrwl/cli/bin/nx.js"
```